signer/local/local.go

```diff
@@ -22,6 +22,11 @@ import (
 	"github.com/cloudflare/cfssl/info"
 	"github.com/cloudflare/cfssl/log"
 	"github.com/cloudflare/cfssl/signer"
+
+	"encoding/asn1"
+	"encoding/binary"
```

Comment:
`go get golang.org/x/tools/cmd/goimports && goimports signer/local/local.go` will fix this.